stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/CallState.kt

```diff
@@ -333,9 +359,22 @@ public class CallState(
 
     /** if we are in backstage mode or not */
     val backstage: StateFlow<Boolean> = _backstage
+
     /** the opposite of backstage, if we are live or not */
     val live: StateFlow<Boolean> = _backstage.mapState { !it }
 
+    /** how many MS the call has been running, null if the call didn't start yet */
```

Comment:
Same as above: Instead of `MS` I would either use `ms` or maybe it will be more clear with `milliseconds`

Reply:
updated the docs.